src/doc_builder/convert_rst_to_mdx.py

```diff
@@ -437,7 +440,11 @@ def parse_rst_docstring(docstring):
             # The line may contain the return type.
             return_type, lines[idx] = split_return_line(lines[idx])
             idx += 1
-            while idx < len(lines) and (is_empty_line(lines[idx]) or find_indent(lines[idx]) >= return_indent):
+            while idx < len(lines) and (
+                is_empty_line(lines[idx])
+                or find_indent(lines[idx]) >= return_indent
+                or _re_raises.search(lines[idx]) is not None
+            ):
```

Comment:
@sgugger let me know if the brackets look good 😊

Reply:
Ah, thanks for checking! It is not ;-)
Here is what we need:
```suggestion
            while idx < len(lines) and_re_raises.search(lines[idx]) is None and (
                is_empty_line(lines[idx])
                or find_indent(lines[idx]) >= return_indent
            ):
```

Reply:
thanks a lot!